### PR TITLE
feat: cronos blockchain icon

### PR DESCRIPTION
**Pull request by art-eth**
Added icon for Cronos chain.

Depends on:
https://github.com/wagmi-dev/references/pull/343

### CODE REVIEW

File: packages/rainbowkit/src/components/RainbowKitProvider/provideRainbowKitChains.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ type ChainName =
   | 'arbitrumGoerli'
   | 'avalanche'
   | 'avalancheFuji'
+  | 'cronos'
```
Comment (DanielSinclair) [inline packages/rainbowkit/src/components/RainbowKitProvider/provideRainbowKitChains.ts:11]: Missing `cronosTestnet`

File: packages/rainbowkit/src/components/RainbowKitProvider/chainIcons/cronos.svg
Comment (DanielSinclair) [inline packages/rainbowkit/src/components/RainbowKitProvider/chainIcons/cronos.svg:1]: Icon is missing background color

**Review by DanielSinclair — CHANGES_REQUESTED** [review]

File: packages/rainbowkit/src/components/RainbowKitProvider/chainIcons/cronos.svg
Comment (DanielSinclair) [reply packages/rainbowkit/src/components/RainbowKitProvider/chainIcons/cronos.svg:1]: Please also minify and match the svg sizing of the other chain icons. This helps bundle sizes.

Comment (0xartcro) [reply packages/rainbowkit/src/components/RainbowKitProvider/chainIcons/cronos.svg:1]: Uploaded new one, let me know if its better now.

> Please also minify and match the svg sizing of the other chain icons. This helps bundle sizes.

**Comment by 0xartcro** [comment]
Looks like `cronosTestnet` is not present in `wagmi` right now. Created PR there as well https://github.com/wagmi-dev/references/pull/343

File: packages/rainbowkit/src/components/RainbowKitProvider/provideRainbowKitChains.ts (hunk 1 above)
Comment (0xartcro) [reply packages/rainbowkit/src/components/RainbowKitProvider/provideRainbowKitChains.ts:11]: Added, depends on https://github.com/wagmi-dev/references/pull/343

**Comment by 0xartcro** [comment]
wagmi@1.2.1 has been released which should contain `cronosTestnet` now.

**Comment by 0xartcro** [comment]
wagmi@1.2.1 has been released which should contain `cronosTestnet` now.

@DanielSinclair This PR is ready to review once again

**Comment by DanielSinclair** [comment]
@0xartcro Awesome, thanks!

**Review by nickbytes — APPROVED** [review]

**Review by DanielSinclair — APPROVED** [review]

**Pull request merged**